mteb/abstasks/TaskMetadata.py

```diff
@@ -180,3 +187,58 @@ def _check_dataset_revision_is_specified(cls, dataset):
                 "It is encourage to specify a dataset revision for reproducability"
             )
         return dataset
+
+    @field_validator("eval_langs")
+    def _check_eval_langs(cls, eval_langs):
+        """
+        This method checks that the eval_langs are specified as a list of languages.
+        """
+        if isinstance(eval_langs, list):
```

Comment:
If someone specifies a tuple, we are not covering. Perhaps have the first check a dict, the rest check sequences?